packages/ethereum-light-client/src/types/sync_committee.rs

```diff
@@ -0,0 +1,124 @@
+use alloy_primitives::Bytes;
+use ethereum_utils::slot::compute_epoch_at_slot;
+use serde::{Deserialize, Serialize};
+use tree_hash_derive::TreeHash;
+
+use super::{
+    bls::{BlsPublicKey, BlsSignature},
+    height::Height,
+    wrappers::WrappedVecBlsPublicKey,
+};
+
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug, Default, TreeHash)]
+pub struct SyncCommittee {
+    pub pubkeys: WrappedVecBlsPublicKey,
+    #[serde(with = "ethereum_utils::base64::fixed_size")]
+    pub aggregate_pubkey: BlsPublicKey,
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug)]
+pub enum ActiveSyncCommittee {
+    Current(SyncCommittee),
+    Next(SyncCommittee),
+}
+
+impl Default for ActiveSyncCommittee {
+    fn default() -> Self {
+        ActiveSyncCommittee::Current(SyncCommittee {
+            pubkeys: WrappedVecBlsPublicKey::default(),
+            aggregate_pubkey: BlsPublicKey::default(),
+        })
+    }
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug, Default)]
+pub struct TrustedSyncCommittee {
+    pub trusted_height: Height,
+    pub current_sync_committee: Option<SyncCommittee>,
+    pub next_sync_committee: Option<SyncCommittee>,
+}
+
+impl TrustedSyncCommittee {
+    pub fn get_active_sync_committee(&self) -> ActiveSyncCommittee {
+        if let Some(sync_committee) = &self.current_sync_committee {
+            ActiveSyncCommittee::Current(sync_committee.clone())
+        } else if let Some(sync_committee) = &self.next_sync_committee {
+            ActiveSyncCommittee::Next(sync_committee.clone())
+        } else {
+            ActiveSyncCommittee::default()
+        }
+    }
```

Comment:
Then this needs to be an enum.

Reply:
It is an enum? But the data is originally coming from the BeaconAPI, so any change to the incoming data would happen as part of #143.

Adding a comment to look more into if this should return default or not.

Reply:
Yes, correct. So that is the Beacon API type that does that. So in #143 we'll have to figure out how to deal with that (probably in the light client still, to avoid making the interface into the light client even more tricky by asking to change the types coming from the Beacon API).